src/home/migrations/0027_manual_paragraph_data.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.10 on 2020-04-02 21:08
+
+from django.db import migrations
+from itertools import chain
+from utils.data_migrations import stream_field_filter_map
+
+def richtext_to_paragraph(block):
```

Comment:
Does this mean that all rich text blocks are converted to normal paragraph blocks?